cartography/intel/azure/logic_apps.py

```diff
@@ -0,0 +1,104 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
```

Comment:
Don't import these. Use `dict` and `list` primitive types available in python 3.9+. Mostly the dicts will be `dict[str, Any]`